controller/frontend/src/Controller/Frontend/Customer/Standard.php

```diff
@@ -92,8 +96,16 @@ public function add( array $values ) : Iface
 			$this->item->setCode( $code );
 		}
 
+		if ( $oldPassword = $values['customer.oldpassword'] ?? null) {
+		    $confirmed = $values['customer.newpassword'] === $values['customer.confirmnewpassword'];
+		    $isNew = $values['customer.newpassword'] !== $values['customer.oldpassword'];
+		    if ($this->item->verifyPassword($oldPassword) && $confirmed && $isNew) {
+		        $this->item = $this->item->setPassword($values['customer.newpassword']);
```

Comment:
There are a few rules that have to be enforced for passwords:
- min 8 chars
- max 64 chars
- includes upper and lower case chars
- includes numbers

Reply:
i implemented the check in ai-client-html, it wont reach this code if the requirements arent meet